src/JabbR-Core/Commands/CommandManager.cs

```diff
@@ -183,32 +199,32 @@ public void MatchCommand(string commandName, out ICommand command)
             }
         }
 
-        private static IList<ICommand> GetCommands()
+        public static IList<ICommand> GetCommands()
         {
             // Use MEF to locate the content providers in this assembly
             //var catalog = new AssemblyCatalog(typeof(CommandManager).Assembly);
             //var compositionContainer = new CompositionContainer(catalog);
             //return compositionContainer.GetExportedValues<ICommand>().ToList();
-            return new List<ICommand>() { new JoinCommand() };
+            return new List<ICommand>() { new JoinCommand(), new OpenCommand(), new CreateCommand(), new LeaveCommand() };
```

Comment:
MatchCommands works, but getCommands does not because we don't have the catalog working how it did in old Jabbr. That's why we have to hard code the commands we are adding in.